mk/config.mk

```diff
@@ -61,6 +61,12 @@ CFG_TEE_TA_LOG_LEVEL ?= 1
 # CFG_TEE_TA_LOG_LEVEL. Otherwise, they are not output at all
 CFG_TEE_CORE_TA_TRACE ?= y
 
+# When "y", the current thread will be shown
+CFG_TRACE_THREAD_ID ?= y
```

Comment:
I remove this in the review commit moving thread id, if we decide to keep that format, then this should also be removed.